internal/pkg/skuba/deployments/ssh/kubernetes.go

```diff
@@ -76,8 +79,23 @@ func kubernetesInstallNodePattern(t *Target, data interface{}) error {
 	if !ok {
 		return errors.New("couldn't access kubernetes base OS configuration")
 	}
+
+	_, _, err := t.ssh("zypper", "--non-interactive", "install", "--force",
+		fmt.Sprintf("patterns-caasp-Node-%s", kubernetes.MajorMinorVersion(version.MustParseSemantic(kubernetesBaseOSConfiguration.KubernetesVersion))),
+		fmt.Sprintf("kubernetes-kubeadm=%s", kubernetesBaseOSConfiguration.KubernetesVersion),
```

Comment:
I've nothing against that, but do we actually need to specify the kubeadm version to be installed? The pattern should already lead to the right version to be selected.

Reply:
if you install the `1.14` pattern, and kubernetes is relaxed to allow kubeadm `1.15` you will always get kubeadm `1.15` here. so we need this otherwise it's impossible to bootstrap a `1.14` cluster

Reply:
updated with a comment